### PR TITLE
Ascent, Conduit & VTK-h: don't assume I have Fortran

### PR DESCRIPTION
**Pull request by ax3l**
Does not assume I have a working Fortran compiler in my toolchain :)

loosely related to #32283

### CODE REVIEW

**Review by kwryankrattiger — DISMISSED** [review]
LGTM

File: var/spack/repos/builtin/packages/vtk-h/package.py [hunk 1]
```diff
@@ -197,7 +197,8 @@ def hostconfig(self):
         if "+mpi" in spec:
             mpicc_path = spec["mpi"].mpicc
             mpicxx_path = spec["mpi"].mpicxx
-            mpifc_path = spec["mpi"].mpifc
+            has_mpifc = hasattr(spec["mpi"], 'mpifc')
```
Comment (ax3l) [inline var/spack/repos/builtin/packages/vtk-h/package.py]:
```suggestion
            has_mpifc = hasattr(spec["mpi"], "mpifc")
```

**Review by cyrush — APPROVED** [review]
Thanks Axel!

**Comment by ax3l** [comment]
My pleasure 🙂

**Pull request merged**